homeassistant/helpers/state.py

```diff
@@ -79,7 +80,8 @@
     SERVICE_LOCK: STATE_LOCKED,
     SERVICE_UNLOCK: STATE_UNLOCKED,
     SERVICE_OPEN_COVER: STATE_OPEN,
-    SERVICE_CLOSE_COVER: STATE_CLOSED
+    SERVICE_CLOSE_COVER: STATE_CLOSED,
+    SERVICE_SET_COVER_POSITION: STATE_SET_COVER_POSITION
```

Comment:
This won't work. The cover component doesn't have a position as state. Instead import and add `ATTR_POSITION` to the other dict above as a value for the `SERVICE_SET_COVER_POSITION` key.